### DISCUSSION TITLE
Mac 本地编译出错

### DISCUSSION THREAD

**Discussion by tanrenxuan** [apache/doris#37696] [RFC]
电脑配置：Apple M3 Pro
环境：按官方文档安装了各种环境
代码版本：master 分支，commit id：5367f7ed8b15f91e63ee5c2ff98de82ec6450742
命令：使用 sh build.sh --be 进行编译。
问题：出现如下错误：
<img width="1227" alt="截屏2024-07-12 10 45 30" src="https://github.com/user-attachments/assets/06f3f0a6-3fb4-4bb1-b7c5-cca9feb35b70">
<img width="1366" alt="截屏2024-07-12 10 47 34" src="https://github.com/user-attachments/assets/f385627e-2821-4fd2-95fc-23f7b2337463">

**Comment by ixzc** [comment]
中文问题在ask.selectdb.com提个帖子

**Comment by qinxike** [comment]
maven并行编译锁问题，重新编译，加上国内的maven仓库跑快点。单阿里云仓库还少包，还要官方的仓库，两个仓库一起跑

**Comment by Andersuc** [comment]
参考这里 https://doris.apache.org/zh-CN/community/developer-guide/mac-dev/dev-prepare
记得依赖先安装下 brew install automake autoconf libtool pkg-config texinfo coreutils gnu-getopt \
python@3 cmake ninja ccache bison byacc gettext wget pcre maven llvm@16 openjdk@17 npm
